src/NServiceBus.RavenDB.Tests/SubscriptionStorage/When_receiving_duplicate_subscription_messages.cs

```diff
@@ -48,14 +68,41 @@ public class When_receiving_duplicate_subscription_messages : RavenDBPersistence
         await storage.Subscribe(subscriber_v6_2, messageType, new ContextBag());
 
         WaitForIndexing(store);
- 
+
         var subscriber = (await storage.GetSubscriberAddressesForMessage(new[]
         {
             messageType
         }, new ContextBag())).ToArray();
- 
+
         Assert.AreEqual(1, subscriber.Length);
         Assert.AreEqual(subscriberAddress, subscriber[0].TransportAddress);
         Assert.AreEqual("new_endpoint_name", subscriber[0].Endpoint);
     }
+
+    [Test]
+    public async Task should_overwrite_existing_subscription_event_if_event_version_changed()
```

Comment:
No, it should test the following scenario:
* Publisher has subscription for `MyMessage v1.0` from subscriber `A`
* Request from subscriber `A` comes in to subscribe to `MyMessage v1.1`
* Publisher overwrites the `v1.0` subscription with the new one.